angular/lib/src/lib/accordion/accordion.component.ts

```diff
@@ -194,21 +200,21 @@ export class AccordionItemComponent implements OnChanges, AfterContentChecked, A
 	/**
 	 * If `true`, the content of the accordion-item collapse will be removed from the DOM. It will be just hidden otherwise.
 	 */
-	@Input('auItemDestroyOnHide') itemDestroyOnHide: boolean | undefined;
+	@Input({alias: 'auItemDestroyOnHide', transform: booleanAttribute}) itemDestroyOnHide: boolean | undefined;
```

Comment:
could it be then undefined now ?

Reply:
As discussed, we will have to use our own transform, to do in another PR.